R/readEurObservER.R

```diff
@@ -44,7 +44,81 @@ readEurObservER <- function(subtype) {
     heatPumpsBarometer.stock = {
 
       # read and join all files
-      files <- c(grep("(20(1[4-9]|2[0-9])_Table3|2013_Table[5-6])\\.csv$", files, value = TRUE))
+      files <- grep("(20(1[4-9]|2[0-9])_Table3|2013_Table[5-6])\\.csv$", files, value = TRUE)
+
+      data <- do.call("rbind", lapply(files, function(file) {
+        reportPeriod <- gsub(".*(\\d{4}).*", "\\1", file)
+        wideTable <- read.csv(file, header = FALSE)
+
+        if (reportPeriod == "2013") {
+          years <- switch(gsub(".*Table|.csv", "", file), `5` = 2011, `6` = 2012)
```

Comment:
```suggestion
          years <- switch(gsub(".*Table|\\.csv", "", file), `5` = 2011, `6` = 2012)
```
The regex also captures (any-character)csv, I'm not sure if that's intended?

Reply:
Right, thanks!